tests/php/src/MobileRedirectionTest.php

```diff
@@ -185,7 +190,8 @@ public function test_redirect_when_server_side_and_not_applicable() {
 		add_filter( 'amp_mobile_client_side_redirection', '__return_false' );
 		add_filter( 'amp_pre_is_mobile', '__return_false' );
 
-		$this->go_to( amp_add_paired_endpoint( '/' ) );
```

Comment:
Going to the AMP endpoint seems to have been a bug here. I'm unclear why it was passing tests before.